docs/api.md

```diff
@@ -406,6 +406,12 @@ if the url has a query parameter of `keys=true`.
   
   <!--end-request-body-post-accountcreate-metricsContext-->
 
+* `marketingOptIn`: *boolean*
+
+  <!--begin-request-body-post-accountcreate-marketingOptIn-->
+  
```

Comment:
I know that much of the documentation is still incomplete but I also reckon that, going forward, we should have a policy of making sure that any new stuff has a description.